test/controllers/LineageExplorerControllerSpec.scala

```diff
@@ -94,7 +92,7 @@ class LineageExplorerControllerSpec extends PlaySpec with OneAppPerSuite with Mo
   }
   "The lineage from context controller" should {
     "return an empty list if the id of the node is not a deployernode" in {
-      val result = lineageController.lineageFromContext( ( "0" ).toLong ).apply( fakerequest )
+      val result = lineageController.lineageFromContext( "0".toLong ).apply( fakerequest )
```

Comment:
`0L` or just `0` doesn't work instead of `"0".toLong`?

Reply:
Good idea, I changed it to 0L.